me.sergiotarxz.JapaChar.yml

```diff
@@ -0,0 +1,99 @@
+id: "me.sergiotarxz.JapaChar"
+branch: "main"
```

Comment:
```suggestion
```

never a flatpak branch

also invalid value.

Reply:
Can I remove this option? I cannot find documentation for that.

Reply:
what's unclear?

Reply:
I do not understand your message, I do not know what a "flatpak branch" is.

I understand every other message you sent, but I am having trouble understanding this.

Reply:
I have deleted the option entirely, I hope that's enough, reopen if it isn't please.

Reply:
that was the suggestion. still don't get what was unclear.

Reply:
Ah, ok, sorry, I did not got that was a suggestion.